assets/runtime/env-defaults

```diff
@@ -101,6 +101,9 @@ GITLAB_WORKHORSE_TIMEOUT=${GITLAB_WORKHORSE_TIMEOUT:-5m0s}
 GITLAB_ARTIFACTS_ENABLED=${GITLAB_ARTIFACTS_ENABLED:-true}
 GITLAB_ARTIFACTS_DIR="${GITLAB_ARTIFACTS_DIR:-$GITLAB_SHARED_DIR/artifacts}"
 
+## Cron Jobs
+GITLAB_PIPELINE_SCHEDULE_WORKER_CRON=${GITLAB_PIPELINE_SCHEDULE_WORKER_CRON:-'0 */12 * * *'}
```

Comment:
The default value for `GITLAB_PIPELINE_SCHEDULE_WORKER_CRON` should read `19 * * * *`. See https://about.gitlab.com/gitlab-com/settings/#cron-jobs